src/components/radio-button/radio-button.js

```diff
@@ -9,8 +10,9 @@ const RadioButton = ({
   handleOptionChange,
   labelText,
   additionalClassName,
+  icon,
 }) => {
-  const labelClassName = `${additionalClassName} min-w-200px bg-stone-200 bg-no-repeat bg-10px p-3 pl-42px ml-1 cursor-pointer hover:bg-neutral-600 hover:text-white peer-checked:bg-neutral-600 peer-checked:text-white peer-focus:bg-neutral-600`;
+  const labelClassName = `${additionalClassName} min-w-200px bg-stone-200 bg-no-repeat bg-10px p-3 ml-1 cursor-pointer hover:bg-neutral-600 hover:text-white peer-checked:bg-neutral-600 peer-checked:text-white peer-focus:bg-neutral-600`;
```

Comment:
You shouldn't need bg-no-repeat anymore, and I don't believe bg-10px is a valid tailwind class so you shouldn't need that either